src/main.js

```diff
@@ -1,14 +1,26 @@
 const Apify = require('apify');
-const { handlGithubIssues } = require('./routes');
-const { getGithubIssuesUrls } = require('./tools');
+const { handleGithubIssues } = require('./issues-scraper');
+const { sendModifiedIssuesNotification } = require('./slack-notifier');
+const { getGithubIssuesRequests, getModifiedIssues } = require('./tools');
+const { ISSUES_STATE } = require('./constants');
 
 const { utils: { log } } = Apify;
 
 Apify.main(async () => {
-    const { repositories, token, channel, proxyConfiguration } = await Apify.getInput();
+    const input = await Apify.getInput();
+    const { repositories, token, separateNotification, proxyConfiguration, excludeOpenedIssues, excludeClosedIssues } = input;
 
-    const issuesUrls = getGithubIssuesUrls(repositories);
-    const requestList = await Apify.openRequestList('github-issues-urls', issuesUrls);
+    // handle missing '#' at the beginning of channel name
+    const channel = input.channel[0] === '#' ? input.channel : `#${input.channel}`;
+
+    const issuesState = await Apify.getValue(ISSUES_STATE) || {};
+    Apify.events.on('persistState', async () => Apify.setValue(ISSUES_STATE, issuesState));
```

Comment:
Add `await` before  `Apify.setValue` mainly for clarity

Reply:
@metalwarrior665 Even when ES Lint complains with: *Redundant use of `await` on a return value* ? 🤔

Reply:
I 
 t

That is because you are returning the Promise but here returning on event doesn't do anything. I'm not 100% sure if this has runtime impact but this better clarifies the intention.

```
 Apify.events.on('persistState', async () => { await Apify.setValue(ISSUES_STATE, issuesState); });
```

Reply:
Ok, I see 👍 ES lint is fine with this version (I just didn't wrap the `setValue` function call into { } block 🙂)